src/test/java/teammates/ui/webapi/GetActionClassesActionTest.java

```diff
@@ -0,0 +1,58 @@
+package teammates.ui.webapi;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.ui.output.ActionClasses;
+
+/**
+ * SUT: {@link GetActionClassesAction}.
+ */
+public class GetActionClassesActionTest extends BaseActionTest<GetActionClassesAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.ACTION_CLASS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Test
+    @Override
+    protected void testExecute() throws Exception {
+        Set<String> actionClasses = new HashSet<>();
+        for (Map<String, Class<? extends Action>> map : ActionFactory.ACTION_MAPPINGS.values()) {
+            for (Class<? extends Action> action : map.values()) {
+                actionClasses.add(action.getSimpleName());
+            }
+        }
```

Comment:
This is just a different way of doing:
```java
ActionFactory.ACTION_MAPPINGS.values().stream()
        .flatMap(map -> map.values().stream().map(Class::getSimpleName))
        .collect(Collectors.toList());
```
i.e. copying the implementation and passing it as the test. So my previous comment will apply here.

Reply:
Understood. This time I have converted back to a list of action classes, but instead of having plain strings in the list, I have the Class object in the list to ensure that it is a valid action class.